api/src/main/java/org/apache/iceberg/types/TypeUtil.java

```diff
@@ -534,6 +534,7 @@ private static int estimateSize(Type type) {
       case FIXED:
         return ((Types.FixedType) type).length();
       case BINARY:
+      case VARIANT:
```

Comment:
What is the rationale for this size?

Reply:
We can't have the accurate size for Variant similar to Binary. So I use the same value as Binary. I'm wondering how we come up with 80 for Binary.